apps/cowswap-frontend/src/common/hooks/useSolversInfo.ts

```diff
@@ -9,13 +9,13 @@ export function useSolversInfo(chainId: SupportedChainId): Record<string, Solver
   const allSolversInfo = useAtomValue(solversInfoAtom)
 
   return useMemo(() => {
-    const env = isProdLike ? 'prod' : 'barn' // Should match what's set on CMS!
+    const envToFilter = isProdLike ? 'prod' : 'staging' // Filters by 'staging' for non-prod (dev/local/"barn") environments because the `solversInfoAtom` data (via CMS mapping) uses 'staging' for these cases.
```

Comment:
If we use 'barn' here, we're not getting any data on our Cosmos Fixture.

Reply:
Won't it break non-cosmos environment?

Reply:
<img width="1217" alt="image" src="https://github.com/user-attachments/assets/fc3c3c45-b94f-49b0-947e-9c28c8aa22cf" />

I see, it's related to this change

Reply:
I'm unsure about this. We should double check. I do know that without this change, it was not getting data in the Cosmos fixture.

Reply:
@alfetopito can you doeble-check please?
I see you added mapping at #5212.
Everything looks legit, just want to be sure.